HW28/src/main/java/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,57 @@
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.List;
+import java.util.ArrayList;
+
+public class CoffeeOrderBoard {
+    private static final Logger logger = LoggerFactory.getLogger(CoffeeOrderBoard.class);
+
+    private List<Order> orders;
+    private int lastOrderNumber;
+
+    public CoffeeOrderBoard() {
+        logger.info("Initializing CoffeeOrderBoard");
+        orders = new ArrayList<>();
```

Comment:
A list on the queue base must be here, not ArrayList - from a performance perspective.